--- a/plugin.cfg
+++ b/plugin.cfg
@@ -66,3 +66,4 @@
 whoami:whoami
 on:github.com/coredns/caddy/onevent
 sign:sign
+unbound:github.com/coredns/unbound
--- a/Makefile.release
+++ b/Makefile.release
@@ -80,6 +80,8 @@
 	@go version
 	@echo Cleaning old builds
 	@rm -rf build && mkdir build
+	@go get github.com/coredns/caddy/onevent
+	@go get github.com/coredns/unbound
 #	@echo Building: darwin/amd64 - $(VERSION)
 #	mkdir -p build/darwin/amd64 && $(MAKE) coredns BINARY=build/darwin/amd64/$(NAME) SYSTEM="GOOS=darwin GOARCH=amd64" CHECKS="" BUILDOPTS=""
 #	@echo Building: windows/amd64 - $(VERSION)
@@ -88,7 +90,7 @@
 #	mkdir -p build/linux/mips  && $(MAKE) coredns BINARY=build/linux/mips/$(NAME) SYSTEM="GOOS=linux GOARCH=mips" CHECKS="" BUILDOPTS=""
 	@echo Building: linux/$(LINUX_ARCH) - $(VERSION) ;\
 	for arch in $(LINUX_ARCH); do \
-	    mkdir -p build/linux/$$arch  && $(MAKE) coredns BINARY=build/linux/$$arch/$(NAME) SYSTEM="GOOS=linux GOARCH=$$arch" CHECKS="" BUILDOPTS="" ;\
+	    mkdir -p build/linux/$$arch  && $(MAKE) gen coredns BINARY=build/linux/$$arch/$(NAME) SYSTEM="GOOS=linux GOARCH=$$arch" CHECKS="" BUILDOPTS="" ;\
 	done
 
 .PHONY: tar
